desktop/desktop.scss

```diff
@@ -231,3 +231,71 @@ and (max-width: 1250px)
     border-top: none;
 }
 
+.widget-button.btn.notifications-dropdown .fa {
+  margin-left: -2px;
+}
+
+// RTL Overrides
+.rtl {
+  .topic-avatar {
+    padding-right: initial;
+    padding-left: 10px;
+    .avatar-flair {
+      right: initial;
+      left: 4px;
+    }
+  }
+
+  .embedded-posts .topic-avatar {
+    padding-left: initial;
+    padding-right: 0;
+  }
+
+  button#login-link, button#new-account-link {
+    padding-left: initial;
+    padding-right: inherit;
+  }
+
+  #create-topic.btn-default {
```

Comment:
I didn't get what you mean there? I don't see anything unusual. Would you be more specific please?